### ISSUE TITLE
A good learning schedule using uPIT for Dense-UNet?

### ISSUE THREAD

**Issue by weedwind** [yuzhou-git/deep-casa#3]
Hi, Yuzhou. Thank you for sharing the scripts. I am training Dense-UNet using uPIT, trying to reproduce the result of 15.2 dB reported in your paper. Did you use the same learning rate schedule (lr=0.0001, multiply by 0.84 if no increase every 6 epochs, and 200 epochs in total, with batch size 1) to get that result?
